tests/formats/zeek/nested-2.yaml

```diff
@@ -0,0 +1,22 @@
+script: |
+  zq -f zeek -o out.zeek in.tzng
```

Comment:
i did it this way because i thought the output was always tzng, but now i see the `output-format` field, will use that and switch to the simpler syntax.